docs/tabs-and-screens/key-req.md

```diff
@@ -37,3 +37,27 @@ By default, this tab is only accessible by users with the following roles:
     Buttons representing possible actions may vary depending on the current status of a certain request at the time of interaction.
 - **Search** bar — allows to search the contents of the **Key Recovery Requests** table.\
   The bar supports letters, numeric values, and dates in the YYYY-MM-DD format (e.g., 2022-10-31).
+
+## Authorizing recovery requests
+
+For the convenience of Bokolo Mobile users, they can restore access to their account by using their unique Recovery QR Code that is automatically generated for every user once they register. However, if a user have lost their Recovery QR Code, they can initiate a manual recovery procedure.
+
+Once a user initiates the procedure to restore access to their account without the Recovery QR Code, their request must be authorized by their bank's PG [Administrator](../roles.md#administrator) or [Authorizer](../roles.md#authorizer).
+
+::: info
+
+For details on how to restore access to a Bokolo Mobile account as a mobile user, see [Help for Bokolo Mobile: Restore access to your account](https://soramitsu.github.io/bokolo-mobile-help/account-operations/restore-account.html).
+
+:::
+
+To authorize a request to restore a Bokolo Mobile account, perform the following:
+
+1. Once a Bokolo Mobile user initiates the procedure, they are prompted to visit their bank's office (see [Help for Bokolo Mobile: Restore access to your account (Step 3)](https://soramitsu.github.io/bokolo-mobile-help/account-operations/restore-account.html#:~:text=If%20you%20don%E2%80%99t%20have%20your%20Recovery%20QR%20Code%3A)) and a new entry appears on the **Key recovery requests** list.
+2. Request from the person that claims to be the user their identification documents and verify whether they are the same person.
+3. From the **Key recovery requests** list select the request that the user submitted.
```

Comment:
It works a little bit differently (at least for now). Here are the steps in case the user doesn't have the QR code: 
1. The user initiates the key recovery process and visits the bank office. 
2. The user provides the documents (passport/ID card). 
3. The PG admin finds the user from the list *Users*. 
4. The PG admin clicks on this user in that list. 
5.  The PG admin clicks the button *Prepare key recovery procedure*
6. A pop-up window appears. The PG admin clicks the button *Continue* and an OTP code is sent to the user's phone number. 
7. The user gives/shows the code to the PG admin. 
8. The PG admin enters the code in the pop-up window and clicks the button *Continue*. 
9. Then the PG admin must enter the user's document ID and attach an application document. 
10. The PG admin clicks the button "Display code". 
11. The user must enter this code and the document ID number in the application. 
12. Then the user appears in the list **Key recovery requests**. The** PG authorizer** approves/rejects the user's request (in the list **Key recovery requests**). 
13. **The Core administrator** approves/rejects the user's request. 
And that's all. 
Then the user continues the recovery process in the app: generates a QR core, enters a password, etc.